app/views/current_optional_subjects/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :credits do %>
+  <%= render 'shared/credits_counter' %>
+<% end %>
+
+<% content_for :welcome_banner do %>
+  <% unless current_student.welcome_banner_viewed? %>
+    <%= render 'shared/welcome_banner' %>
```

Comment:
i wonder if we should render the banner or not. I think right now we only do it in the home page, right?

Reply:
Removed